<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1383</width>
    <height>935</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="filters">
       <attribute name="title">
        <string>Filters</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>10</y>
          <width>531</width>
          <height>541</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Input Image</string>
         </property>
        </widget>
        <widget class="QLabel" name="inputImageLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>512</width>
           <height>511</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_2">
        <property name="geometry">
         <rect>
          <x>750</x>
          <y>10</y>
          <width>531</width>
          <height>541</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>101</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Output Image</string>
         </property>
        </widget>
        <widget class="QLabel" name="outputImageLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>511</width>
           <height>512</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="loadImageBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>131</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>650</y>
          <width>1031</width>
          <height>131</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="widget" native="true">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>20</y>
           <width>411</width>
           <height>91</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Select Filter</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Fixed</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="filterSelect">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <item>
             <property name="text">
              <string/>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Average</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Gaussian</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Median</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Sobel</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Prewitt</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Roberts</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="additionalParmsFrame">
         <property name="geometry">
          <rect>
           <x>519</x>
           <y>39</y>
           <width>431</width>
           <height>71</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
        <widget class="QLabel" name="label_5">
         <property name="geometry">
          <rect>
           <x>520</x>
           <y>20</y>
           <width>161</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Additional Prameteres</string>
         </property>
        </widget>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>630</y>
          <width>111</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Filter Settings</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="imageName">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>130</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
       <widget class="QLabel" name="imageSize">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>170</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="histograms">
       <attribute name="title">
        <string>Histograms</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_13">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,3,3,3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QPushButton" name="LoadImageBtn_1">
                <property name="text">
                 <string>Load Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="imageSelect">
                <item>
                 <property name="text">
                  <string>GrayImage</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RGBImage</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Input Image</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="InputImagLabel_1">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>Histogram</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="InputImagLabel_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>CDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="InputImagLabel_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="3,3,3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>OutputImage</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="outpuImageLabel_1">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_14">
                <property name="text">
                 <string>Histogram</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="outputImageLabel_2">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7" stretch="0,1">
              <item>
               <widget class="QLabel" name="label_15">
                <property name="text">
                 <string>CDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="outputImageLabel_3">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="frequencyFilters">
       <attribute name="title">
        <string>Frequency Filters</string>
       </attribute>
       <widget class="QPushButton" name="plotBtn">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>270</y>
          <width>89</width>
          <height>25</height>
         </rect>
        </property>
        <property name="text">
         <string>plot</string>
        </property>
       </widget>
       <widget class="QFrame" name="customPlot">
        <property name="geometry">
         <rect>
          <x>240</x>
          <y>40</y>
          <width>581</width>
          <height>531</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="hybrid">
       <attribute name="title">
        <string>Hybrid</string>
       </attribute>
       <widget class="QFrame" name="frame_4">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>20</y>
          <width>411</width>
          <height>401</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_7">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Image A</string>
         </property>
        </widget>
        <widget class="QLabel" name="imageALabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>391</width>
           <height>371</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QFrame" name="frame_5">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>430</y>
          <width>411</width>
          <height>401</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_9">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Image B</string>
         </property>
        </widget>
        <widget class="QLabel" name="imageBLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>391</width>
           <height>371</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="imageABtn">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Image A</string>
        </property>
       </widget>
       <widget class="QPushButton" name="imageBBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>490</y>
          <width>151</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Image B</string>
        </property>
       </widget>
       <widget class="QLabel" name="imageASize">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>200</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="imageAName">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>160</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>200</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="imageBSize">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>620</y>
          <width>161</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="imageBName">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>580</y>
          <width>141</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>620</y>
          <width>41</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Size:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>580</y>
          <width>51</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Name:</string>
        </property>
       </widget>
       <widget class="QPushButton" name="hybridBtn">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>740</y>
          <width>131</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>Make Hybrid</string>
        </property>
       </widget>
       <widget class="QFrame" name="frame_6">
        <property name="geometry">
         <rect>
          <x>790</x>
          <y>110</y>
          <width>531</width>
          <height>541</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QLabel" name="label_14">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>121</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Hybrid Image</string>
         </property>
        </widget>
        <widget class="QLabel" name="hybridImageLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>512</width>
           <height>511</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1383</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
